src/Resources/config/cache.xml

```diff
@@ -65,5 +66,15 @@
         <service id="sonata.cache.invalidation.simple" class="Sonata\Cache\Invalidation\SimpleCacheInvalidation">
             <argument type="service" id="logger"/>
         </service>
+        <service id="sonata.cache.command.flush" class="Sonata\CacheBundle\Command\CacheFlushCommand">
+            <argument type="constant">null</argument>
```

Comment:
This will be handled as `null` or as a string containing the word "null"?

Reply:
As null (I tested it because I barely remembered it from somewhere , but could not find docs about this)